README.md

```diff
@@ -28,24 +28,28 @@ bugsnag:
 # Usage #
 After the installation the bundle works without any additional settings required, but you can tweak some settings.
 
-## Notify Stages ##
-You can set for which environments you want Bugsnag to get error reports. This is done with the notify_stages setting:
+## Enabled Stages ##
+You can set for which environments you want Bugsnag to be enabled. This is done through the enabled_stages setting:
 
 ```yml
 bugsnag:
-    notify_stages: [development, staging, production]
+    enabled_stages: [dev, prod, staging]
```

Comment:
Certainly, however shouldn't we use just `dev` in the example because a standard Symfony setup has a `dev` and `prod` environment, no `development` or `production` environment? It's just us who hate these abbreviated environment names, I bet 9 out of 10 users leave these as-is :)

It might confuse users to mix and match these variations, with the enabled_stages and notify_stages being two separate things (Symfony's `kernel.environment` versus the Bugsnag stage as determined by `ReleaseStage`).

What do you think?